.github/workflows/tests.yml

```
@@ -40,13 +40,19 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: get repository name
+        shell: bash
```

I force push to add this line.
```suggestion
        shell: bash
```